### ISSUE TITLE
Creating MAUI Handler from Avalonia and replacing maui native handlers

### ISSUE THREAD

**Issue by AswinPG** [AvaloniaUI/AvaloniaMauiHybrid#5]
Is it possible to create handlers for maui Controls using Avalonia controls instead of native handlers? 
If that is possible then wouldn't it be like all the work is already done for maui to have a drawn backend ?
All we need to do would be to create mappers that maps the Maui IView's property to Avalonia UI controls right ? 

@jsuarezruiz

**Comment by giuseppenovielli** [comment]
Yes with one single handlers, for example a MAUI.Button or MAUI.StackLayout with Avalonia Handlers, MAUI can support all Avalonia platform, i think for example browsers, Linux and also old Windows 7.
https://avaloniaui.net/Platforms

Avalonia could be embedded into MAUI thanks to MAUI Architecture.
https://github.com/dotnet/maui/discussions/62
https://github.com/dotnet/maui/issues/4528
https://github.com/dotnet/maui/discussions/339
https://github.com/dotnet/maui/discussions/3051

**Comment by ToolmakerSteve** [comment]
I have thought of doing that as an *interim* prototype.  
It is one way to target SkiaSharp, and use SkiaSharp to get to different platforms.

But that adds another UI abstraction into the technology stack.  
I am skeptical of doing so, in the long term.  
But as a starting point, absolutely.  
Then examine each control's implementation. See if there are changes that would align the implementation directly with Maui's features.  

Especially important is how **layouts**, and the control hierarchy, work.  
Definitely do not want to have a second hierarchy (Avalonia's) underneath Maui's.  
Not saying anything bad about Avalonia; just bad to have dual hierarchies, both doing roughly the same work.  

BOTTOM LINE: I am skeptical about "use Avalonia as a target"; better to clone-and-modify individual Avalonia controls, to implement Maui functionality.

-----------------------------------
I'd like to mention an alternative approach. At least to consider the technique; far less mature than Avalonia:

[DrawnUI for .NET MAUI](https://github.com/taublast/DrawnUi.Maui).  
This successfully takes the major Maui UI classes, and renders to a SkiaSharp canvas.  
[Nick Kovalsky - The Quest To Create Drawn .NET MAUI Apps](https://taublast.github.io/posts/MauiJuly/).  
[YouTube - Javier Suarez - DrawnUi.Maui: Drawn Controls using SkiaSharp for .NET MAUI](https://www.youtube.com/watch?v=g38f8otTT3k).  
[Surf App Comparison](https://github.com/taublast/SurfAppCompareDrawn).  

I prefer this approach, because the only UI hierarchy is Maui's.  

Understand that DrawnUI is not currently a "drop in" solution.  
You have to explicitly refer to its class names in your xaml.  
And you explicitly add a "canvas" at the root of your xaml.  
And there are undoubtedly missing features.

But what is there, seems to work well. As an experiment, I modified a simple SkiaSharp-based game to use it for some UI.  
Needs a few pioneers to use it, push its limits, improve it.  

(I was hoping to use it in a commercial project. But the client wants HTML forms. So their non-developers can create and modify the look of the forms. So we go with Maui Blazor.)
